src/librustc_data_structures/indexed_vec.rs

```diff
@@ -103,6 +103,11 @@ impl<I: Idx, T> IndexVec<I, T> {
     }
 
     #[inline]
+    pub unsafe fn set_len(&mut self, len: usize) {
```

Comment:
Turns out I had added it and ended up never removing. I still feel this is a useful function to have for what is essentially a wrapper for the `Vec`. As can be seen from changes to this file, I ended up adding a number of wrapper functions which I used at some point while experimenting, and then forgot to delete. I feel like other people who like experimenting might also find this useful at some point.